samples/RazorSlices.Samples.WebApp/Program.cs

```diff
@@ -30,6 +31,19 @@
     Results.Extensions.RazorSlice("/Slices/LoremInjectableProperties.cshtml", new LoremParams(paraCount, paraLength), serviceProvider));
 app.MapGet("/unicode", () => Results.Extensions.RazorSlice("/Slices/Unicode.cshtml"));
 app.MapGet("/library", () => Results.Extensions.RazorSlice("/Slices/FromLibrary.cshtml"));
+app.MapGet("/render-to-string", () =>
+{
+    var slice = RazorSlice.Create("/Slices/LoremFormattable.cshtml", new LoremParams(1, 4));
+    string template = slice.Render();
+    return Results.Ok(new { HtmlString = template });
+});
```

Comment:
See comment further down regarding this method needing to be async.

```suggestion
app.MapGet("/render-to-string", async () =>
{
    var slice = RazorSlice.Create("/Slices/LoremFormattable.cshtml", new LoremParams(1, 4));
    var template = await slice.RenderAsync();
    return Results.Ok(new { HtmlString = template });
});
```